applications/calculator/main.cpp

```diff
@@ -25,148 +25,191 @@ enum CALCULATOR_OPERATOR
     OPERATOR_CLEAR_ALL = 6
 };
 
-void update_calculator_text()
+#define CALC_ID_BUTTON_0 "0"
+#define CALC_ID_BUTTON_1 "1"
+#define CALC_ID_BUTTON_2 "2"
+#define CALC_ID_BUTTON_3 "3"
+#define CALC_ID_BUTTON_4 "4"
+#define CALC_ID_BUTTON_5 "5"
+#define CALC_ID_BUTTON_6 "6"
+#define CALC_ID_BUTTON_7 "7"
+#define CALC_ID_BUTTON_8 "8"
+#define CALC_ID_BUTTON_9 "9"
+#define CALC_ID_BUTTON_PLUS "plus"
+#define CALC_ID_BUTTON_MINUS "minus"
+#define CALC_ID_BUTTON_EQUAL "equal"
+#define CALC_ID_BUTTON_CLEAR "clear"
+#define CALC_TEXT_BOX_NAME "calcscreen"
+
+class calculator_system
```

Comment:
This is not a calculator_system, it's Calculator please call a spade a spade ;)

Reply:
@Supercip971 looks like you didn't see this message

Reply:
ow yes sorry

Reply:
It's my fault I didn't include this message in the review :sweat_smile:

Reply:
it is now fixed